docs/cloud/cloud-org/cloud-org-billing.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Manage billing
```

Comment:
Manage billing for what?

Manage database billing
Manage organization billing

Pick one and embrace the clarity of communication :D